rov/json_test_files/generate_rov_json.py

```diff
@@ -0,0 +1,39 @@
+import json
```

Comment:
why are these and all other things in `rov/json_test_files`? Can this all be merged into a `tests` folder. I like the idea of feeding through data to test things out, but can this all be combined together to run tests for 1) individual parts 2) full ROV